app/create_app.py

```diff
@@ -59,6 +63,9 @@ def create_app() -> Flask:
 
     logging.init_app(flask_app)
 
+    # Initialise Redis Magic Links Store
+    redis_toggles.init_app(flask_app)
```

Comment:
Looks like you are missing the code to get this to find the credentials on PaaS?

Reply:
its just in draft atm, we are testing the library locally